src/js/GlobalConstants.js

```diff
@@ -18,7 +18,8 @@ const globalConstants = {
     QAT: (process.env.ENV === 'qat' || process.env.ENV === 'sandbox'),
     STAGING: (process.env.ENV === 'staging'),
     FILES_SERVER_BASE_URL: filesServerUrlByEnv[process.env.ENV],
-    ARP_RELEASED: process.env.ENV !== 'prod'
+    ARP_RELEASED: process.env.ENV !== 'prod',
+    AGENCY_LINK: process.env.ENV === 'prod' ? 'agency' : 'agency_v2'
```

Comment:
Based on [this change](https://github.com/fedspendingtransparency/usaspending-website/pull/2790/files#L25) in your other PR, I think we'll want a second constant tied to the agency v2 release (`ARP_RELEASED` is for the Covid profile) or to change the logic here slightly. We need some way to ensure that `agency/{slug}` links don't show up in prod _yet_.

Reply:
Yep, my bad, thanks!